app/config.js

```diff
@@ -0,0 +1,222 @@
+    format: String,
+    default: "cif",
+    nullable: false,
+    env: "DATABASE_SCHEMA",
+  },
+  sitewideNotice: {
+    doc: "An html notice to print at the top of the page, e.g. to identify dev/test versions of the app.",
+    format: String,
+    nullable: true,
+    default: null,
+    env: "SITEWIDE_NOTICE",
+  },
+  supportEmail: {
+    doc: "Support email address provided.",
+    format: "email",
+    default: "ggircs@gov.bc.ca",
+    env: "SUPPORT_EMAIL",
+  },
+  sentryEnvironment: {
+    doc: "",
+    format: String,
+    default: "localhost",
+    env: "SENTRY_ENVIRONMENT",
+  },
+  sentryRelease: {
+    doc: "",
+    format: String,
+    default: "local_development",
+    env: "SENTRY_RELEASE",
+  },
+  // renamed from ENABLE_DB_MOCKS
+  enableMockTime: {
+    doc: "Show a calendar on screen to allow time travel in -dev and -test environments.",
+    format: "Boolean",
+    default: false,
+    env: "ENABLE_MOCK_TIME",
+  },
+  enableDbMocksCookiesOnly: {
+    doc: "Allows a cookie to be set to mock the time (no calendar).",
+    format: "Boolean",
+    default: false,
+    env: "ENABLE_DB_MOCKS_COOKIES_ONLY",
+  },
+  storageApiHost: {
+    doc: "Host of storage API.",
+    format: "host",
+    default: "http://localhost:8000",
+    env: "STORAGE_API_HOST",
+  },
+  storageApiKey: {
+    // Not yet used in deployments, so no validation function yet.
+    doc: "Api key allowing access to storage api, needs to match what's in storage api env.",
+    default: null,
+    env: "STORAGE_API_KEY",
+  },
+  enableMockAuth: {
+    doc: "",
+    default: null,
+    format: "Boolean",
+    nullable: true,
+    env: "ENABLE_MOCK_AUTH",
+  },
+  mockAuthCookie: {
+    default: "mocks.auth",
+    format: String,
+    env: "MOCK_AUTH_COOKIE",
+  },
+  showKCLogin: {
+    doc: "Show the keycloak login page.",
+    default: false,
+    format: "Boolean",
+    env: "SHOW_KC_LOGIN",
+  },
+  cifRole: {
+    doc: "Login as a specific role.",
+    nullable: true,
+    default: null,
+    format: ["CIF_INTERNAL", "CIF_EXTERNAL", "CIF_ADMIN", "UNAUTHORIZED_IDIR"],
+    arg: "as-role",
+  },
+  happoApiKey: {
+    default: null,
+    format: "*",
+    env: "HAPPO_API_KEY",
+  },
+  happoApiSecret: {
+    default: null,
+    format: "*",
+    env: "HAPPO_API_SECRET",
+  },
+});
+
+// In case default host is used with non-default port
+if (
+  config.get("host") === "http://localhost:3004" &&
+  config.get("port") !== 3004
+) {
+  config.load({ host: `http://localhost:${config.get("port")}` });
+}
+
+if (config.get("namespace").endsWith("-prod") && config.get("enableMockTime")) {
+  throw new Error("ENABLE_MOCK_TIME cannot be true with a -prod namespace.");
+}
+
+if (config.get("namespace") && config.get("enableMockAuth")) {
+  throw new Error("ENABLE_MOCK_AUTH cannot be true when NAMESPACE is set.");
+}
+
+if (config.get("namespace") && config.get("cifRole")) {
+  throw new Error("Cannot set both namespace and cifRole.");
+}
+
```

Comment:
we should throw an error if `namespace` is set but `process.env.SESSION_SECRET` isn't, as this would lead to sessions being attached to the pod a request is hitting